packages/flutter/test/material/filled_button_test.dart

```diff
@@ -239,6 +239,7 @@ void main() {
 
   testWidgets('Default FilledButton meets a11y contrast guidelines', (WidgetTester tester) async {
     final FocusNode focusNode = FocusNode();
+    addTearDown(() => focusNode.dispose());
```

Comment:
nit: I think this can just be `addTearDown(focusNode.dispose)`.

Same for similar occurrences in this PR that just call a single method.